pkg/http/api_directmessaging.go

```diff
@@ -292,3 +313,14 @@ type invokeError struct {
 func (ie invokeError) Error() string {
 	return fmt.Sprintf("invokeError (statusCode='%d') msg='%v'", ie.statusCode, string(ie.msg))
 }
+
+type codeError struct {
+	statusCode  int
+	msg         []byte
+	headers     invokev1.DaprInternalMetadata
+	contentType string
+}
+
+func (ce codeError) Error() string {
+	return fmt.Sprintf("CodeError: received non-successful status code (statusCode='%d') msg='%v'", ce.statusCode, string(ce.msg))
```

Comment:
Errors normally start with lowercased characters in Go.

```suggestion
	return fmt.Sprintf("received non-successful status code in response: statusCode='%d' msg='%v'", ce.statusCode, string(ce.msg))
```